api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -71,4 +70,55 @@
 
     Long getUserCount();
   }
+
+  @Query(
+      "SELECT\n"
+          + "  u.aboutYou,\n"
+          + "  u.areaOfResearch,\n"
+          + "  u.betaAccessBypassTime,\n"
+          + "  u.betaAccessRequestTime,\n"
+          + "  u.complianceTrainingBypassTime,\n"
+          + "  u.complianceTrainingCompletionTime,\n"
+          + "  u.complianceTrainingExpirationTime,\n"
+          + "  u.contactEmail,\n"
+          + "  u.creationTime,\n"
+          + "  u.currentPosition,\n"
+          + "  u.dataAccessLevel,\n"
+          + "  u.dataUseAgreementBypassTime,\n"
+          + "  u.dataUseAgreementCompletionTime,\n"
+          + "  u.dataUseAgreementSignedVersion,\n"
+          + "  u.demographicSurveyCompletionTime,\n"
+          + "  u.disabled,\n"
+          + "  u.emailVerificationBypassTime,\n"
+          + "  u.emailVerificationCompletionTime,\n"
+          + "  u.emailVerificationStatus,\n"
+          + "  u.eraCommonsBypassTime,\n"
+          + "  u.eraCommonsCompletionTime,\n"
+          + "  u.eraCommonsLinkExpireTime,\n"
+          + "  u.familyName,\n"
+          + "  u.firstRegistrationCompletionTime,\n"
+          + "  u.firstSignInTime,\n"
+          + "  u.freeTierCreditsLimitDaysOverride,\n"
+          + "  u.freeTierCreditsLimitDollarsOverride,\n"
+          + "  u.givenName,\n"
+          + "  u.idVerificationBypassTime,\n"
+          + "  u.idVerificationCompletionTime,\n"
+          + "  u.lastModifiedTime,\n"
+          + "  u.organization,\n"
+          + "  u.phoneNumber,\n"
+          + "  u.professionalUrl,\n"
+          + "  u.twoFactorAuthBypassTime,\n"
+          + "  u.twoFactorAuthCompletionTime,\n"
+          + "  u.userId,\n"
+          + "  u.username,\n"
+          + "  a.city,\n"
+          + "  a.country,\n"
+          + "  a.state,\n"
+          + "  a.streetAddress1,\n"
+          + "  a.streetAddress2,\n"
+          + "  a.zipCode\n"
+          + "FROM DbUser u\n"
+          + "  LEFT OUTER JOIN DbAddress AS a ON u.userId = a.user.userId\n"
```

Comment:
This join is easy to forget. We agreed to export information on active workspaces only, as inactive workspaces aren't a fully-defined, user-facing feature.